package.json

```diff
@@ -63,11 +63,10 @@
     "@angular/compiler-cli": "^17.0.2",
     "@compodoc/compodoc": "^1.1.22",
     "@ngneat/spectator": "^15.0.1",
-    "@storybook/addon-a11y": "^7.5.3",
-    "@storybook/addon-docs": "^7.5.1",
+    "@storybook/addon-a11y": "^7.6.3",
+    "@storybook/addon-docs": "^7.6.3",
     "@storybook/addon-essentials": "^7.5.3",
     "@storybook/addon-interactions": "^7.5.3",
-    "@storybook/addon-interactions": "^7.5.1",
```

Comment:
I saw a PR that recently removed this duplicate line 🤔 

https://github.com/LuccaSA/lucca-front/pull/2439/files

Maybe just merge master into rc once the PR is merged ?